src/hotspot/share/code/nmethod.cpp

```diff
@@ -341,28 +341,24 @@ void ExceptionCache::set_next(ExceptionCache *ec) {
 // Helper used by both find_pc_desc methods.
 static inline bool match_desc(PcDesc* pc, int pc_offset, bool approximate) {
   NOT_PRODUCT(++pc_nmethod_stats.pc_desc_tests);
-  if (!approximate)
+  if (!approximate) {
     return pc->pc_offset() == pc_offset;
-  else
+  } else {
     return (pc-1)->pc_offset() < pc_offset && pc_offset <= pc->pc_offset();
+  }
 }
 
-void PcDescCache::reset_to(PcDesc* initial_pc_desc) {
-  if (initial_pc_desc == nullptr) {
-    _pc_descs[0] = nullptr; // native method; no PcDescs at all
-    return;
-  }
-  NOT_PRODUCT(++pc_nmethod_stats.pc_desc_resets);
-  // reset the cache by filling it with benign (non-null) values
-  assert(initial_pc_desc->pc_offset() < 0, "must be sentinel");
-  for (int i = 0; i < cache_size; i++)
+void PcDescCache::init_to(PcDesc* initial_pc_desc) {
```

Comment:
I renamed method to `init_to()` because it is used only once for initialization.